.github/workflows/terraform-lint.yaml

```diff
@@ -36,6 +36,11 @@ jobs:
         uses: actions/checkout@v3
 
       - name: tfsec
+        uses: aquasecurity/tfsec-action@v1.0.3
```

Comment:
`aquasecurity/tfsec-pr-commenter-action` runs tfsec itself, so this isn't needed.

See https://github.com/aquasecurity/tfsec-pr-commenter-action/blob/7a44c5dcde5dfab737363e391800629e27b6376b/entrypoint.sh#L66

Reply:
Huh -- I missed that while reviewing the tfsec-pr-commenter-action while trying to figure out why we weren't seeing results.

Still a bit mysterious; we'll work on trivy instead and close this.